libs/langchain-baidu-qianfan/src/chat_models.ts

```diff
@@ -217,9 +215,9 @@ export class ChatBaiduQianfan
 
   userId?: string;
 
-  modelName = "ERNIE-Bot-turbo";
+  modelName = "ERNIE-Lite-8K";
 
-  model = "ERNIE-Bot-turbo";
+  model = "ERNIE-Lite-8K";
```

Comment:
Just in case someone is relying on this behavior, let's not update the default model. Updating docs is fine.

We've been moving away from hidden defaults and towards requiring a model string in general.

Reply:
I have remove the update of  default model and added abandoned model info in docs. Please check these changes, thanks.